psi4/src/export_fock.cc

```diff
@@ -95,7 +95,9 @@ void export_fock(py::module &m) {
         .def("wK", &JK::wK, py::return_value_policy::reference_internal)
         .def("D", &JK::D, py::return_value_policy::reference_internal)
         .def("computed_shells_per_iter", &JK::computed_shells_per_iter, "Array containing the number of ERI shell quartets computed (not screened out) during each compute call.")
-        .def("print_header", &JK::print_header, "docstring");
+        .def("print_header", &JK::print_header, "docstring")
+        .def("incfock_last_iter", &JK::incfock_last_iter, "Was the last Fock build incremental?")
+        .def("reset_incfock", &JK::reset_incfock, "Reset the incfock iteration number (0 means no incfock)");
```

Comment:
According to `jk.h`, `incfock_count_` means "The number of times INCFOCK has been performed (includes resets)". If so, why are we ever resetting this? Does 0 mean no incfock has ever been performed, or that we should not perform incfock in future?

Reply:
An `incfock_count_` of 0 means that this is the first SCF iteration, so no incfock should be performed. In the context of `COSX`, the reason why this variable is reset to 0 is to avoid incremental Fock builds with the last iteration (where the COSK algorithm is performed on a larger grid). Incfock is NEVER performed at iteration 0.

Reply:
Okay. Then I think it would be clearer if instead of setting `incfock_iter_` to 0 (changing the meaning of the keyword), you have some dedicated flag to disable incfock on specific iterations.

Reply:
Okay I can do that

Reply:
Done, I added a new keyword called `disable_incfock_`